src/pages/Tools/Timer/TimeSettingModal/index.tsx

```diff
@@ -0,0 +1,153 @@
+import { ChangeEventHandler, useState } from 'react';
+
+import useModal from '@Hooks/useModal';
+
+import localStorageHelper from '@Utils/localStorageHelper';
+
+import Button from '@Components/Button';
+import Input from '@Components/Input';
+
+import * as S from './style';
+
+type Props = {
+  changeInitTime: (minute: number, second: number) => void;
+};
+
+function TimeSettingModal({ changeInitTime }: Props) {
+  const { closeModal } = useModal();
+
+  const recentTimes = localStorageHelper.get<number[]>('timer') ?? [];
+  const recentTime = recentTimes[0] ?? 300;
+
+  const [minute, setMinute] = useState(Math.floor(recentTime / 60));
+  const [second, setSecond] = useState(recentTime - minute * 60);
+
+  const handleChangeTime: (
+    type: 'minute' | 'second',
+  ) => ChangeEventHandler<HTMLInputElement> = (type) => (event) => {
+    const value = Number(event.target.value);
+
+    if (isNaN(value)) return;
+    if (minute === 60 && type === 'second') {
+      setSecond(0);
+      return;
+    }
+
+    if (value === 60 && type === 'minute') {
+      setMinute(60);
+      setSecond(0);
+      return;
+    }
+
+    if (value > 60 && type === 'minute') {
+      setMinute(60);
+      return;
+    }
```

Comment:
```suggestion
    if (value >= 60 && type === 'minute') {
      setMinute(60);
      setSecond(0);
      return;
    }
```
맥시멈이 60분이면 요렇게 묶어야 하는 거 아닌가요? ㅎㅎ 아님 제가 이해를 잘 못했을 수도..

Reply:
맞아요..!! 같이 묶는게 더 좋네요ㅎㅎ
조건을 이리저리 생각하다 조건문을 더 사용한 듯 해요